Makefile

```diff
@@ -32,6 +32,7 @@ all:
 	@echo "link:       Runs 'python setup.py --develop' in all subprojects and links the needed resources"
 	@echo "clean:      Get rid of scratch, byte files and removes the links to other subprojects"
 	@echo "selfcheck:  Runs tree static check, unittests and functional tests using Avocado itself"
+	@echo "spell:      Runs spell checker on comments and docstrings (requires python-enchant)"
```

Comment:
Shouldn't we include this in `make check`? And if so then we should add `pyenchant` to `requirements-selftests`.